readme-vars.yml

```diff
@@ -25,10 +25,14 @@ param_usage_include_env: true
 param_env_vars:
   - { env_var: "TZ", env_value: "Europe/London", desc: "Specify a timezone to use EG Europe/London." }
   - { env_var: "MAX_UPLOAD", env_value: "5000", desc: "To set maximum upload size (in MB), default if unset is 5000." }
+  - { env_var: "DB_HOST", env_value: "<yourdbhost>", desc: "for specifying the database host" }
+  - { env_var: "DB_USER", env_value: "<yourdbuser>", desc: "for specifying the database user" }
+  - { env_var: "DB_PASS", env_value: "<yourdbpass>", desc: "for specifying the database password" }
+  - { env_var: "DB_DATABASE", env_value: "projectsend", desc: "for specifying the database to be used (non-alphanumeric passwords must be properly escaped.)" }
+
 param_usage_include_vols: true
 param_volumes:
-  - { vol_path: "/config", vol_host_path: "<path to data>", desc: "Where to store projectsend config files." }
-  - { vol_path: "/data", vol_host_path: "<path to data>", desc: "Where to store files to share." }
+  - { vol_path: "/data", vol_host_path: "/path/to/data", desc: "Where to store files to share." }
```

Comment:
it needs to be /config, there is no /data. I just meant change the description, the original /config description says projectsend, it needs to be more generic since you're showing the same for both now, like <https://github.com/linuxserver/docker-bookstack/blob/675eb4ecbf528c76073678ff8614af209a949c22/readme-vars.yml#L25>